### PR TITLE
Fix iconUrl warning

### PR DESCRIPTION
**Pull request by fredericDelaporte**
Fix the `iconUrl` warning occurring on NuGet push.
See https://docs.microsoft.com/en-us/nuget/reference/msbuild-targets#packageiconurl

### CODE REVIEW

**Comment by hazzik** [comment]
I would be using [`Directory.Build.props`](https://docs.microsoft.com/en-us/visualstudio/msbuild/customize-your-build) technique to solve this.

**Comment by fredericDelaporte** [comment]
> I would be using [`Directory.Build.props`](https://docs.microsoft.com/en-us/visualstudio/msbuild/customize-your-build) technique to solve this. 

Thanks for the pointer. I have used the common props we already have instead, since we can `Remove` where needed.

File: appveyor.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 version: 5.7.0.{build}
-image: Visual Studio 2017
+image: Visual Studio 2019
```
Comment (fredericDelaporte) [inline appveyor.yml:2]: Checking the built artifacts, the `icon` declaration is missing in packages .nuspec. I suspect it requires a VS2019 setup.

**Comment by hazzik** [comment]
I’ll try something to see if it works (in next 2 hours)

**Comment by fredericDelaporte** [comment]
With the VS-2019 image, it works.

**Comment by hazzik** [comment]
I've made the icon included conditionally and project path independent.

**Pull request merged**